util/memory/action.go

```diff
@@ -43,8 +47,7 @@ func (a *LogOnExceed) Action(t *Tracker) {
 	defer a.mutex.Unlock()
 	if !a.acted {
 		a.acted = true
-		logutil.Logger(context.Background()).Warn("memory exceeds quota",
```

Comment:
How about check if `logHook` is nil, and if it is we leave this log as a default action.